### ISSUE TITLE
Original and revised budgets added up

### ISSUE THREAD

**Issue by KasperBrandt** [akvo/akvo-rsr#1505]

**Comment by lauraroverts** [comment]
E.g. 

In the admin I filled in the original budget with an amount, period start- and period end date. I also filled in the revised budget with an amount, and a start date. See screenshot.
![screen shot 2015-04-22 at 15 41 35](https://cloud.githubusercontent.com/assets/4179989/7275780/176f6c1e-e906-11e4-8b91-495ff47c803c.png)

If you look at http://rsr.akvo.org/project/2893/finance/ I would expect to see an overview of the history of original and revised budgets.  The revised budget would be the latest and thus up to date budget. However, original and revised budgets added up. As a result, it looks like the project is still looking for funding, because the amount that is funded bij the Dutch Ministry of Foreign, which is the total revised budget,is only part of the total of the original + revised budget. 

Also, the begin- and end date I filled in in the admin are not visible at the overview. And it is not clear what the project's financial status is: fully funded vs needs funding. 

Might be that part of the problem is due to the fact that in the admin it is not possible to add a 'Other 1' and 'Total' budget item at the same time.
